### ISSUE TITLE
ember-cli-shims error

### ISSUE THREAD

**Issue by marcoow** [ember-cli/ember-cli-blueprint-test-helpers#33]
I'm seeing the ember-cli-shims-versions-prior-to-0.1.0 error when running Ember Simple Auth's blueprint tests with Ember/Ember Data/Ember CLI 2.4.x:

```
1) Acceptance: ember generate and destroy authenticator generates an OAuth 2.0 authenticator:
     Error: the object {
  "errorLog": [
    {
      "isSilentError": true
      "message": "Using a version of ember-cli-shims prior to 0.1.0 will cause errors while loading Ember Data 2.3.0-beta.3+. Please update ember-cli-shims from undefined to 0.1.0."
      "name": "SilentError"
      "suppressedStacktrace": true
    }
  ]
  "exitCode": 1
  "inputStream": []
  "outputStream": []
  "statusCode": 1
} was thrown, throw an Error :)
```

I'm not sure whether that's a bug in this library or something wrong with the way it's being used in ESA but I was under the impression that bower is skipped in the temp project that the blueprints are run in anyway so that it shouldn't have `ember-cli-shims` at all?

This pull request has the code that updates Ember CLI et al in ESA with the failing tests: https://github.com/simplabs/ember-simple-auth/pull/918

**Comment by trabus** [comment]
It seems you're missing your test package fixtures, I'd try running the default blueprint again `ember g ember-cli-blueprint-test-helpers`

**Comment by trabus** [comment]
@marcoow some investigation brought me to this line in ember-data: https://github.com/emberjs/data/blob/f261ca8902219244994e79ba4785607d845d53ce/index.js#L33

and running the node test with `EMBER_DATA_SKIP_VERSION_CHECKING_DO_NOT_USE_THIS_ENV_VARIABLE=true npm run nodetest`
makes the error go away, and fixes all the tests except for the throws tests:
```js
  Acceptance: ember generate and destroy authenticator
    ✓ generates a torii authenticator (669ms)
    ✓ generates an OAuth 2.0 authenticator (220ms)
    ✓ generates a devise authenticator (187ms)
    ✓ generates a generic authenticator (171ms)
Authenticators cannot extend from themself. Remove the --base-class option or specify one of "oauth2", "torii" or "devise".
    1) throws when the authenticator is specified as its own base class
The authenticator base class "unknown" is unknown. Remove the --base-class option or specify one of "oauth2", "torii" or "devise".
    2) throws when an unknown base class is specified

  Acceptance: ember generate and destroy authorizer
    ✓ generates an OAuth 2.0 authorizer (185ms)
    ✓ generates a devise authorizer (207ms)
    ✓ generates a generic authorizer (178ms)
Authorizers cannot extend from themself. Remove the --base-class option or specify one of "oauth2" or "devise".
    3) throws when the authorizer is specified as its own base class
The authorizer base class "unknown" is unknown. Remove the --base-class option or specify one of "oauth2" or "devise".
    4) throws when an unknown base class is specified```

For the throws issue, it seems that this change: https://github.com/ember-cli/ember-cli/commit/d79f810df80b851e099e7678eeb1ce20c85b74f4#diff-046dd933d1aae8e9131812190820a4c6R81
broke the assertThrows check, because we are expecting the error rather than the whole results object:
https://github.com/ember-cli/ember-cli-blueprint-test-helpers/blob/master/lib/helpers/blueprint-helper.js#L245

I will file an issue for the broken assertThrows functionality and try and get a new release cut soon.

**Comment by rwjblue** [comment]
damn, and I thought `EMBER_DATA_SKIP_VERSION_CHECKING_DO_NOT_USE_THIS_ENV_VARIABLE` was long enough  ;P

**Comment by trabus** [comment]
I'm probably not going to file the 'throws' issue tonight, I'm laying in bed sick, so I'll probably try and pick it back up tomorrow. Something is definitely funky with it though, We may need to revisit how we're doing it here.

**Comment by marcoow** [comment]
Awesome @trabus, thanks for finding that!

**Comment by trabus** [comment]
@marcoow reported the throws issue in #34 and will be fixed when #35 is merged.

**Comment by marcoow** [comment]
@trabus: yay - thanks for that!

**Comment by trabus** [comment]
@marcoow just released v0.10.0 which should fix throws assertions for you. Thanks for your patience!
